packages/honeypot/tsconfig.cjs.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": "../../tsconfig.cjs.json",
+    "compilerOptions": {
+        "rootDir": "./src",
+        "outDir": "./dist/cjs",
+        "lib": ["es6", "dom"],
+        "jsxImportSource": "@emotion/react"
```

Comment:
Is this required? @emotion/react is not used in your code.

Reply:
Sorry, I didn't removed it after test
I will remove these